jsapp/scss/components/_kobo.form-view.scss

```diff
@@ -892,6 +898,68 @@
   }
 }
 
+.form-modal__form.form-modal__form--data-attachment-columns {
+   color: $kobo-gray-50;
+
+  .bulk-options {
+    margin-top: 14px;
+    display: flex;
+    justify-content: space-between;
+
+    .bulk-options__description {
+      font-weight: bold;
+    }
+
+    .bulk-options__buttons {
+      span {
+        margin: 12px;
+      }
+
+      a {
+        text-decoration: underline;
+        cursor: pointer;
+      }
+    }
+  }
+
+  .multi-checkbox {
+    margin-top: 12px;
+    height: 200px;
+  }
+
+  .loading {
+    margin-top: 12px;
+  }
+
+  .modal__footer {
+    text-align: center;
+
+    button {
+      padding-left: 64px;
+      padding-right: 64px;
+    }
+  }
+
+}
+
+// Compensate for when sidebar(s) messes up modal a bit
+@media
```

Comment:
Instead of all that, just go with `modal--large` on modal :) I think it will be quite useful for real life projects, where question names are loooooong

Reply:
Ok I tested "just go with" and it will require a bit more work. The `--large` modal is flexible due to `height: 80%`, so the `.form-modal__form--data-attachment-columns` would need to be creative with `flex` so the multi checkbox is also flexible and takes always all possible vertical space (with some minimum height of course)

Reply:
Just let me know if you need some help with this, as I see by using dev tools it will be a bit tricky